hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestNameNodeRpcServer.java

```diff
@@ -1,4 +1,4 @@
-/**
+/*
```

Comment:
?

Reply:
Those license headers aren't javadoc and shouldn't start with the two stars. If you load it in intellij, they show up as a warning. Since I was fixing the imports, I figured I might as well fix that too.